gradle/verification-metadata.xml

```diff
@@ -641,6 +639,11 @@
             <pgp value="31bae2e51d95e0f8ad9b7bcc40a3c4432bd7308c"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.15.1">
+         <artifact name="gradle-enterprise-gradle-plugin-3.15.1.jar">
+            <pgp value="7b79add11f8a779fe90fd3d0893a028475557671"/>
+         </artifact>
+      </component>
```

Comment:
❌ I am somewhat surprised by this. Is there a new signing key for the Gradle Enterprise plugin? If yes, it should be updated in the trusted key entry above.

Also, can we remove the `com.gradle` entries now from the trusted key `1bd97a6a154e7810ee0bc832e2f38302c8075e3d` above? If we can't, were do we have artifacts that use the old signing key?

Reply:
Hi @wolfs, thanks very much for your review. Afaiu we're indeed signing the GE plugin with a new key. I've removed the old `com.gradle` entries, and moved the `gradle-enterprise-gradle-plugin` key to a trusted key. Please let me know if this looks fine to you?